src/data/qpf/multivariate/basic.lean

```diff
@@ -0,0 +1,216 @@
+/-
+Copyright (c) 2018 Jeremy Avigad. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Author: Jeremy Avigad
+-/
+import data.pfunctor.multivariate.basic
+universe u
+
+/-!
+# Multivariate quotients of polynomial functors.
+
+Basic definition of multivariant QPF
```

Comment:
Could you please expand the docstring? E.g., when I read this I can't understand whether this is going to be useful for me.

Reply:
Is this more helpful?

Reply:
Yes!

Reply:
I believe it might be a bit far from analysis, it might not be that useful to you

Reply:
Now I understand that this is some generic way to deal with (co)inductive types. And yes, now I immediately see that this is probably not that useful for me.